packages/load-script/src/dom-operations.js

```diff
@@ -3,14 +3,26 @@ import { handleRequestError, handleRequestSuccess } from './callback-handler';
 
 const debug = debugFactory( 'lib/load-script/dom-operations' );
 
-export function createScriptElement( url ) {
+export function createScriptElement( url, args ) {
 	debug( `Creating script element for "${ url }"` );
 	const script = document.createElement( 'script' );
+
 	script.src = url;
 	script.type = 'text/javascript';
 	script.async = true;
 	script.onload = handleRequestSuccess;
 	script.onerror = handleRequestError;
+
+	if ( args ) {
+		Object.entries( args ).forEach( ( [ key, value ] ) => {
+			if ( ! value ) {
+				script.removeAttribute( key );
+			} else {
+				script.setAttribute( key, value );
+			}
```

Comment:
That should do the job as well, and it seems to pass the tests:

```suggestion
			script[ key ] = value;
```

Reply:
Alternatively, we could set `async` to `true` only if `async` is not already set by `args`. Worth trying IMO.